README.md

```diff
@@ -2,6 +2,12 @@
 
 V-pipe is a workflow designed for clinical applications of next generation sequencing (NGS) data to viral pathogens. It produces a number of results in a curated format.
 
+[![Snakemake](https://img.shields.io/badge/snakemake-≥4.8.0-brightgreen.svg?style=plastic)](https://snakemake.bitbucket.io)
+[![bio.tools](https://img.shields.io/badge/bio-tools-orange.svg?style=plastic)](https://bio.tools/V-Pipe)
```

Comment:
Can you change the contact information to the V-pipe mail?

Reply:
modifications force-pushed

Reply:
The two comments above were concerning the bio.tools website